bot.js

```diff
@@ -65,110 +65,68 @@ if (!process.env.clientId || !process.env.clientSecret || !process.env.PORT) {
   // process.exit(1);
 }
 
-var Botkit = require('botkit');
-var debug = require('debug')('botkit:main');
+const Botkit = require('botkit');
+const debug = require('debug')('botkit:main');
 
-var bot_options = {
+const bot_options = {
     clientId: process.env.clientId,
     clientSecret: process.env.clientSecret,
     clientSigningSecret: process.env.clientSigningSecret,
     // debug: true,
     scopes: ['bot'],
     studio_token: process.env.studio_token,
-    studio_command_uri: process.env.studio_command_uri
+    studio_command_uri: process.env.studio_command_uri,
+    require_delivery: true, // required for RTM (Real Time Messaging) support
```

Comment:
This is required if we ever use RTM, it makes sure a message is delivered before the bot actions on it and ensures things happen in order.